### PR TITLE
Add files to correct platform targets

### PR DESCRIPTION
**Pull request by onmyway133**
- This checks, fixes and re-adds files to correct platform targets

### CODE REVIEW

File: Source/iOS/UIImage+Extensions.swift [hunk 1]
```diff
@@ -23,4 +19,11 @@ extension UIImage {
 
     return result
   }
+
+  /// Convert to data
+  func cache_toData() -> Data? {
```
Comment (vadymmarkov) [inline Source/iOS/UIImage+Extensions.swift]: Why do we need `cache` prefix here? It's not even public.

Comment (onmyway133) [reply Source/iOS/UIImage+Extensions.swift]: It's not public, but it can cause conflicts 😬

Comment (onmyway133) [reply Source/iOS/UIImage+Extensions.swift]: I don't like underscore TBH

Comment (vadymmarkov) [reply Source/iOS/UIImage+Extensions.swift]: If it's not public I think it's fine to go without prefix.

**Pull request merged**